project_setup/README.md

```diff
@@ -29,35 +28,34 @@ branch protection - [see our branch protection guide for details](branch-protect
 - [Setting up branch protection](#setting-up-branch-protection)
 - [Setting up type-specific configuration settings](#setting-up-type-specific-configuration-settings)
   - [Setting up Coveralls for Python projects](#setting-up-coveralls-for-python-projects)
-  - [Ansible requirement file generation tool 🧻🛠](#ansible-requirement-file-generation-tool-)
-  - [Terraform IAM credentials to GitHub secrets 🔑‍👉🤫](#terraform-iam-credentials-to-github-secrets-)
-  - [Managing SSM parameters from files 🗂👉☁️](#managing-ssm-parameters-from-files-%EF%B8%8F)
+  - [Ansible requirement file generation tool](#ansible-requirement-file-generation-tool)
+  - [Terraform IAM credentials to GitHub secrets](#terraform-iam-credentials-to-github-secrets)
+  - [Managing SSM parameters from files](#managing-ssm-parameters-from-files)
 
 ## Adding a skeleton configuration to an existing repository ##
 
 To skeletonize an existing repository, please see the guide to
 [skeletonize an existing repository](skeletonize-existing-repository.md).
 
-## Using the skeleton tool to start a new repository 💀🛠 ##
+## Using the gh-skeleton `gh` extension to start a new repository ##
 
 We recommend that you start your project from one of the
 [skeleton projects](https://github.com/search?q=org%3Acisagov+topic%3Askeleton)
-in this organization.  The [`skeleton`](scripts/skeleton)
-helper tool included in the [`scripts`](scripts) directory can quickly setup
-a new local repository.  Once you've cloned and configured the repository
-to your local machine, you then publish it to a repository created on GitHub.
+in this organization.  The [`gh-skeleton`][gh-skeleton]
+`gh` extension provides the ability to easily start new projects from the existing
+library of skeleton repositories.
 
 ### Selecting a skeleton ###
 
 First, identify a suitable skeleton project to use as the starting point
 for your new repository. For a list of available skeletons, see the
 [Skeleton List](skeleton-list.md) or use the following command:
 
-```bash
-./skeleton list
-```
-
 ```console
+$ gh skeleton list
+
+
+
```

Comment:
As long as we are updating this section, we should also update to show the current correct output:

```console
$ gh skeleton list
Available skeletons in cisagov:

skeleton-ansible-role    	A skeleton project for quickly getting a new cisagov Ansible role started.
skeleton-ansible-role-with-test-user	A skeleton project for quickly getting a new cisagov Ansible role started when that role requires an AWS test user.
skeleton-aws-lambda-python	A skeleton project for quickly getting a new Python-based AWS Lambda project started.
skeleton-docker          	A skeleton project for quickly getting a new cisagov Docker container started.
skeleton-generic         	A generic skeleton project for quickly getting a new cisagov project started.
skeleton-packer          	A skeleton project for quickly getting a new cisagov packer project started.
skeleton-python-library  	A skeleton project for quickly getting a new cisagov Python library started.
skeleton-tf-module       	A skeleton project for quickly getting a new cisagov Terraform module started.
```

Reply:
Hi @dav3r, update to output for gh skeleton list. Addressed at 3443b94

Reply:
@dv4harr10 Why did you replace the whitespace between the skeleton repo names and their descriptions (in my sample output above) with newlines in https://github.com/cisagov/development-guide/commit/3443b94a4c2e3967ecd5407277d69abf1578accb?  Our formatting inside the `console` block should look the same as the output that users will see in their terminal.

Reply:
Hi @dav3r, the whitespace was giving a linter error. I can try to include it back.

Reply:
Can you please show us exactly what error message you were getting?

Reply:
Hi @dav3r , it's resolved with 2be365c

Reply:
The error message was trim trailing whitespace, but it's now resolved.

Reply:
Unfortunately, https://github.com/cisagov/development-guide/commit/2be365cadb2870773571224402ed0f7ae20c2320 still does not address my original concern:

> Why did you replace the whitespace between the skeleton repo names and their descriptions (in my sample output above) with newlines in https://github.com/cisagov/development-guide/commit/3443b94a4c2e3967ecd5407277d69abf1578accb?

Reply:
Hi @dav3r , please give me a call, I posted my number to the chat Thanks

Reply:
Hi @dav3r, update to output for gh skeleton list should be resolved at 8513d28

Reply:
@dv4harr10 Looks good now with https://github.com/cisagov/development-guide/pull/97/commits/8513d287209b2f0c4ab81d68af244759eea753b9, thanks!